docs/practices/security_vulnerabilities.md

```diff
@@ -0,0 +1,46 @@
+## Security Vulnerabilities
+
+<blockquote style="background: rgba(255, 200, 0, 0.1); border: 5px solid rgba(255, 200, 0, 0.4);">
+The intended audience of the information presented here are developers working
```

Comment:
Worth checking if this needs a whitespace between tags and contents. IIRC markdown needs a space to generate a `<p>` for the text and/or to interpret html-looking stuff as html, but that may just be github-flavoured markdown.

Reply:
But we can also just fix that afterwards if its broken too.

Reply:
Good idea.  It appeared fine in my local setup.  I'll double check once it is merged.

Reply:
The render on current master is at https://near.github.io/nearcore/practices/security_vulnerabilities.html.  It looks fine to me, so I suppose we are fine here.